include/openPMD/auxiliary/StringManip.hpp

```diff
@@ -183,5 +203,34 @@ join(std::vector< std::string > const& vs, std::string const& delimiter)
             return ss.str();
     }
 }
+
+inline std::string
+removeSlashes( std::string s )
```

Comment:
please add a doxygen string: e.g. explain where in the string slashes are removed